tests/includes/API/Actions/CountActionTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Waca\API\Actions\CountAction;
+
+class CountActionTest extends PHPUnit_Framework_TestCase
+{
+	private $count;
+
+	public function setUp() {
+		$count = new CountAction();
+
+		$this->count = $count;
+
+		$this->assertInstanceOf(CountAction::class, $count);
```

Comment:
I'm fairly sure this is guaranteed to pass by the PHP runtime.

(It is PHP though, so who knows...)